Core/scripts/dirac-install.py

```diff
@@ -1895,7 +2008,8 @@ def createCshrcForDiracOS():
       logDEBUG("No dirac-deploy-scripts found. This doesn't look good")
   else:
     logNOTICE("Skipping installing DIRAC")
-  if cliParams.diracOS or cliParams.diracOSVersion:
+
+  if cliParams.diracOS and cliParams.diracOSVersion:
```

Comment:
this is not correct. It should be:
```python
if cliParams.diracOS:
```
Because when you have DiracOS in the release.cfg file we want to use that version:
v6r20-pre16
  {
    Modules = DIRAC, VMDIRAC:v2r2, RESTDIRAC:v0r5, COMDIRAC:v0r17, WebAppDIRAC:v3r1p4
    Externals = v6r6p7
    DiracOS = 0.1
  }
so the command should be:
dirac-install v6r20-pre16 -g v14r0 --dirac-os (which will not work in the current implementation)
if we want to overwrite the version which is in releases.cfg, the command should be:
dirac-install v6r20-pre16 -g v14r0 --dirac-os --dirac-os-version 0.2

Reply:
If I do `dirac-install v6r20-pre16 -g v14r0 --dirac-os`,  I want diracos to be installed, and the version should be taken from the releases.cfg. Is it not what is going to happen ?

Reply:
I think there is a misunderstanding here. We need two information related to diracos. 
1) Do we really want to install it ? 
2) What version do we want to install 

For point 2), we either take what is in releases.cfg, or what is specified by --dirac-os-version.

For point 1), this can only be triggered by a flaf, `--dirac-os`, which is false by default.

Or are you telling me that what is in releases.cfg is not put at the same place than `--dirac-os-version` ?

Reply:
cliParams.diracOSVersion is filled when you use --dirac-os-version.
As I said before you need to use:
```python
if cliParams.diracOS:
```
If --dirac-os-version is provided, will use that version otherwise use the version from release.cfg